### ISSUE TITLE
Contrast check: red on dark blue (accessibility)

### ISSUE THREAD

**Issue by mthmulders** [mthmulders/shades-of-grey#204]
Got feedback at JCON 2024 that the "accent" colour (`text-red-600` in Tailwind) does not have a good contrast with the "background" colour (`bg-gray-900` in Tailwind).

An ideal contrast ration would be 4.5 to 1. This [Contrast Checker](https://webaim.org/resources/contrastchecker/) might be a good tool to verify that.

**Comment by mthmulders** [comment]
# Current

||Background|Text|Accent|
|---|---|---|---|
||`#111827`|`#EEEEEE`|`#DC2626`|
|Contrast Ratio||15.28:1|3.67:1|

||Normal Text|Large Text|Graphical Objects and User Interface Components|
|---|---|---|---|
|WCAG AA|Fail|Pass|Pass|
|WCAG AAA|Fail|Fail||

# Proposed

||Background|Text|Accent|
|---|---|---|---|
||`#111827`|`#EEEEEE`|`#DFC12A`|
|Contrast Ratio||15.28:1|9.96:1|

||Normal Text|Large Text|Graphical Objects and User Interface Components|
|---|---|---|---|
|WCAG AA|Pass|Pass|Pass|
|WCAG AAA|Pass|Pass||

**Comment by Bukama** [comment]
Hey @mthmulders,
as said this week the red was hardly readable on top of the black background when watching the presentation even for me who has no visual impairment and I'm happy that your take up my feedback to improve the layout.

I mentioned the contrast of `4.5:1` as it's the ration of the WCAG to fulfill AA level.
The "European Accessibility Act" (and therefor all the national laws which introduced the EAA in the specific countries) as the EAA literally demands the AA level. Aiming to pass AAA level is always a good idea, but can be tricky at some points.

As you have already listed in your tables the current (red) accent color does not fulfill the WCAG AA requirement for normal text. With your proposed color you even pass the requirements for AAA level, so that would be a good fit from ratio point of view.

If you like the color and/or if it's fits into the whole color layout of the presentation is (obviously ;) ) up to you. From what I have in mind from the slides the proposed color could fit well.

**Comment by mthmulders** [comment]
Hmm, `#DFC12A` is not a color in Tailwind. The closest to it seems `amber-400` or `#fbbf24`.

# Proposed (2)

||Background|Text|Accent|
|---|---|---|---|
||`#111827`|`#EEEEEE`|`#FBBF24`|
|Contrast Ratio||15.28:1|10.62:1|

||Normal Text|Large Text|Graphical Objects and User Interface Components|
|---|---|---|---|
|WCAG AA|Pass|Pass|Pass|
|WCAG AAA|Pass|Pass||

It even has a slightly higher contrast ratio, so that's good!
